e2fsprogs: expand @mkdir_p@

Add AC_SUBST to configure.ac.  @mkdir_p@ is currently
not expanded so no locale data is written into usr/share/locale. 

Upstream-Status: Pending

Signed-off-by: Joe Slater <jslater@windriver.com>

--- a/configure.ac
+++ b/configure.ac
@@ -811,6 +811,8 @@ AC_SUBST(PACKAGE)
 AC_SUBST(VERSION)
 
 AM_GNU_GETTEXT
+dnl @MKDIR_P@ is expanded in AM_GNU_GETTEXT
+AC_SUBST([mkdir_p],['$(MKDIR_P)'])
 dnl
 dnl End of configuration options
 dnl
